pywatts/modules/models/profile_neural_network.py

```diff
@@ -109,7 +110,8 @@ def transform(self, **kwargs) -> xr.DataArray:
 
     def fit(self, target, **kwargs):
         """
-        Fit the Profile Neural Network.
+        Fit the Profile Neural Network. Note historical_input, trend, and profile has to be provided.
```

Comment:
Fit the Profile Neural Network. Note that historical_input, trend, and profile have to be provided.